Specs/DateTime/English/DateTimeModel.json

```diff
@@ -20627,5 +20627,85 @@
         }
       }
     ]
+  },
+  {
+    "Input": "The project was submitted last month and 3 weeks later it was approved",
```

Comment:
Can you add the same/similar specs in other languages too?

Reply:
added in German, Italian and Spanish